docs/maps/connect-to-ems.asciidoc

```diff
@@ -35,3 +35,189 @@ To disable EMS, change your <<settings, kibana.yml>> file.
+
+| `ssl.supportedProtocols:`
+ | An array of supported protocols with versions.
+Valid protocols: `TLSv1`, `TLSv1.1`, `TLSv1.2`, `TLSv1.3`. *Default: `TLSv1.1`, `TLSv1.2`, `TLSv1.3`*. <<server-ssl-supportedProtocols,Equivalent {kib} setting>>.
+
+| `ssl.cipherSuites:`
+ | Details on the format, and the valid options, are available via the
+https://www.openssl.org/docs/man1.1.1/man1/ciphers.html#CIPHER-LIST-FORMAT[OpenSSL cipher list format documentation].
+*Default: `TLS_AES_256_GCM_SHA384 TLS_CHACHA20_POLY1305_SHA256 TLS_AES_128_GCM_SHA256 ECDHE-RSA-AES128-GCM-SHA256, ECDHE-ECDSA-AES128-GCM-SHA256, ECDHE-RSA-AES256-GCM-SHA384, ECDHE-ECDSA-AES256-GCM-SHA384, DHE-RSA-AES128-GCM-SHA256, ECDHE-RSA-AES128-SHA256, DHE-RSA-AES128-SHA256, ECDHE-RSA-AES256-SHA384, DHE-RSA-AES256-SHA384, ECDHE-RSA-AES256-SHA256, DHE-RSA-AES256-SHA256, HIGH,!aNULL, !eNULL, !EXPORT, !DES, !RC4, !MD5, !PSK, !SRP, !CAMELLIA`*. <<server-ssl-cipherSuites,Equivalent {kib} setting>>.
+|===
+
+[float]
+[[elastic-maps-server-bind-mount-config]]
+===== Bind-mounted configuration
+
+One way to configure {hosted-ems} is to provide `elastic-maps-server.yml` via bind-mounting. With +docker-compose+, the bind-mount can be specified like this:
+
+["source","yaml",subs="attributes"]
+--------------------------------------------
+version: '2'
+services:
+  {hosted-ems}:
+    image: {ems-docker-image}
+    volumes:
+      - ./elastic-maps-server.yml:/usr/src/app/config/elastic-maps-server.yml
+--------------------------------------------
+
+[float]
+[[elastic-maps-server-envvar-config]]
+===== Environment variable configuration
+All configuration settings can be overridden by environment variables that are named with the following conditions: all letters are uppercase and replace YAML periods by underscores. For example `elasticsearch.ssl.certificate` could be overridden by the environment variable `ELASTICSEARCH_SSL_CERTIFICATE`. Boolean variables need to use the `true` or `false` strings.
+
+WARNING: All information that you include in environment variables is visible through the `ps` command, including sensitive information.
+
+These variables can be set with +docker-compose+ like this:
+
+["source","yaml",subs="attributes"]
+----------------------------------------------------------
+version: '2'
+services:
+  {hosted-ems}:
+    image: {ems-docker-image}
+    environment:
+      ELASTICSEARCH_HOST: http://elasticsearch.example.org
+      ELASTICSEARCH_USERNAME: 'ems'
+      ELASTICSEARCH_PASSWORD: 'changeme'
+----------------------------------------------------------
+
+[float]
+[[elastic-maps-server-data]]
+==== Data
+
+Boundaries:: World countries, global administrative regions, and countries regions datasets are available.
+
+Basemaps:: A minimal basemap is provided with {hosted-ems}. This can be used for testing environments but won't be functional for normal operations. The full basemap (around 90GB file) needs to be mounted on the image for {hosted-ems} to run normally.
```

Comment:
```suggestion
Basemaps:: A minimal basemap is provided with {hosted-ems}. This can be used for testing environments, but won't be functional for normal operations. The full basemap (around 90GB file) must be mounted on the image for {hosted-ems} to run normally.
```